src/components/Editor/Runners/HtmlRunner/HtmlRunner.js

```diff
@@ -79,13 +78,13 @@ function HtmlRunner() {
 
   const eventListener = () => {
     window.addEventListener("message", (event) => {
-      if (typeof event.data === "string" || typeof event.data === "string") {
-        if (event.data === "ERROR: External link") {
+      if (event.data?.msg && typeof event.data?.msg === "string") {
```

Comment:
aren't you getting the first part of this clause for free by using `?.msg` or am I missing a nuance here?

Reply:
yes!! Ha, left over from playing around with different data shapes :laugh: